vcd/datasource_vcd_nsxv_firewall.go

```diff
@@ -107,16 +107,15 @@ func datasourceVcdNsxvFirewallRule() *schema.Resource {
 								Type: schema.TypeString,
 							},
 						},
-						// TODO - ipsets and security groups need further investigation and at least
-						// "Get" capability in govcd
-						// "ipsets": {
-						// 	Type:        schema.TypeSet,
-						// 	Computed:    true,
-						// 	Description: "Set of IP set names",
-						// 	Elem: &schema.Schema{
-						// 		Type: schema.TypeString,
-						// 	},
-						// },
+						"ipsets": {
```

Comment:
Somehow it didn't poke the eyes in the separate resource `vcd_ipset`, but now it does. By looking at this PR it just begs to be called `ip_sets` instead of `ipsets`. How is API naming/capitalising them?

Reply:
API does not have much of naming. The actual field is `value` while whole structure is `ipset`. I can change. Do you also want to change resource name to `vcd_ip_set`?

Reply:
Yes, if change the field then change the resource as well. For me personally `ip_set` with underscore is clearer and easier to scan/notice than `ipset`. @vbauzysvmware what's your preference?

Reply:
IMHO, I would agree on `ip_set` as in UI we have `New IP Set`. Also I recommend to be more precise with resource name and name it `vcd_edgegateway_ip_set` as it will also consistent with current edge gtw driven resources

Reply:
Changed field to be `ip_sets` and resource/data source to be called `vcd_nsxv_ip_set`